powershell.el

```diff
@@ -1387,6 +1396,78 @@ This insures we get and display the prompt."
 ;;           (insert " "))
 ;;       success)))
 
+(defun powershell--fetch-json-array (url)
```

Comment:
You might just use 
```elisp
(with-temp-buffer (url-insert-file-contents url)
		  (json-parse-buffer :array-type 'list))
```
here instead of using the low-lowel function. 		  `

This also avoids the HTTP-response buffer leftover.

Reply:
Good suggestion. Fixed!